course/src/main/java/org/openedx/course/presentation/unit/html/HtmlUnitFragment.kt

```diff
@@ -114,14 +105,15 @@ class HtmlUnitFragment : Fragment() {
                             HTMLContentView(
                                 windowSize = windowSize,
                                 url = blockUrl,
-                                cookieManager = edxCookieManager,
+                                cookieManager = viewModel.cookieManager,
+                                isLoading,
+                                injectJSList,
```

Comment:
Please add parameter name as well.

Reply:
@HamzaIsrar12 done, and I fixed a bug when unable to load Survey